packages/synapse-react-client/src/components/SynapseTable/ExpandCollapseButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import IconSvg from '../IconSvg'
+
+type ExpandCollapseButtonProps = {
+  isExpanded: boolean
+} & React.ButtonHTMLAttributes<HTMLButtonElement>
+
+export default function ExpandCollapseButton(props: ExpandCollapseButtonProps) {
```

Comment:
Was just used by `ExpandableTableDataCell` but is now used by multiple components, so pull it out as its own component.